lib/workers/branch/index.ts

```diff
@@ -518,7 +519,7 @@ export async function processBranch(
         }
         const context = `renovate/artifacts`;
         const description = 'Artifact file update failure';
-        const state = 'failure';
```

Comment:
This got change from failure to failed, which I'm guessing may not have been deliberate.